lightly/active_learning/scorers/classification.py

```diff
@@ -0,0 +1,31 @@
+from typing import *
+
+import numpy as np
+
+from lightly.active_learning.scorers.scorer import Scorer
+
+
+def entropy(probs: np.ndarray, axis: int = 1):
+    zeros = np.zeros_like(probs)
+    log_probs = np.log2(probs, out=zeros, where=probs > 0)
+    entropies = -1 * np.sum(probs * log_probs, axis=axis)
+    return entropies
+
+
+class ScorerClassification(Scorer):
```

Comment:
Some docstrings here would be helpful. Is this a scorer for "image classification"? Will there be another one for object detection?

Reply:
Yes, there will be another one for object detection in the future, but first we need to define the formats of the prediction of an  object detection model  which will be supported.

Reply:
I created an issue for it here: https://github.com/lightly-ai/lightly/issues/137